documentation/asciidoc/topics/ref_default_cache_service_config.adoc

```diff
@@ -1,11 +1,20 @@
 [id='default_config-{context}']
-= Cache Configuration
+= Default Cache Configuration
 
 {cacheservice} nodes use the following cache configuration:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-include::config_examples/cache_service_default_cache.xml[]
+<distributed-cache name="default" <1>
+                   mode="SYNC" <2>
+                   owners="1"> <3>
+   <memory>
+    <off-heap eviction="MEMORY" <4>
+              strategy="REMOVE"/> <5>
+   </memory>
+   <partition-handling when-split="ALLOW_READ_WRITES" <6>
+                       merge-policy="REMOVE_ALL"/> <7>
+</distributed-cache>
 ----
```

Comment:
@rigazilla should we update this to reflect the new memory configuration in 11?

Reply:
@oraNod sounds good. We also need to update the `owners` attribute which now 2 by default if not specified in `infinispan.spec.service.replicationFactor`

Reply:
@rigazilla hopefully we're good now. Does `max-size="<maximum_size_in_bytes>"` look good? IIRC the operator calculates the maximum size when it creates pods.

Reply:
@oraNod yep you're right

Reply:
great!